.github/ISSUE_TEMPLATE/new_feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: New Feature
+about: new feature proposal for Terracognita
+title: ''
+labels: "Feature"
+assignees: ''
+---
+### Abstract
+<!-- (Required) Only a few lines that should be understandable by everyone -->
+
+### Description
+
+#### Proposal
+<!-- (Required) What do you propose to add to Terracognita -->
+
+#### Implementation
+<!-- (Optional) May be filled later after discussion with the maintainers -->
```

Comment:
Which is the difference between `Description`  and `Proposal` and `Abstract`? and basically the first 2.

Reply:
`Description` is the main title actually. You don't write anything in this section.

`Abstract` is the high level definition: "be able to split the resources following the cloud services"

`Proposal` would be a more "detailed" description .

but it's certainly not clear enough if you ask aha

Reply:
Then I would remove the `Description`, no need for it. I would leave just the other 3 but even with that It feels it's "too much" but let's go with that for now and see.